client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/.eslintignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
I don't think you need this. However, I'm not expecting you to correct all issues in this file.